pytest_localstack/__init__.py

```diff
@@ -9,6 +10,24 @@
 from pytest_localstack import constants, plugin, session, utils
 from pytest_localstack._version import __version__  # noqa: F401
 
+start_timeout = None
+stop_timeout = None
+
+
+def pytest_configure(config):
+    global start_timeout, stop_timeout
+    if config.getoption("--no-localstack"):
+        pm = config.pluginmanager
+        pm.unregister(name="pytest-localstack")
+        warnings.warn(
+            "You can disable this plugin with pytest -p no:localstack",
```

Comment:
I've changed to use the standard naming convention of `pytest_{pluginname}` so that you can now disable the plugin / get the plugin using plugin manager with just the name `localstack`.